session.go

```diff
@@ -215,6 +194,32 @@ func NewSession(cfg ClusterConfig) (*Session, error) {
 	return s, nil
 }
 
+func (s *Session) startReconnection(itvl time.Duration) {
+	go func() {
```

Comment:
No problem.
I've overwritten the last commit. 2ea86d3ae9baa67e82490af4c8caaf622f76794c